extensions/machine-learning-services/src/modelManagement/registeredModelService.ts

```diff
@@ -133,29 +123,57 @@ export class RegisteredModelService {
 	 * @param databaseName
 	 * @param tableName
 	 */
-	private configureTable(databaseName: string, tableName: string): string {
-		let escapedTableName = utils.doubleEscapeSingleBrackets(tableName);
-		let escapedDbName = utils.doubleEscapeSingleBrackets(databaseName);
+	private configureTable(): string {
+		let databaseName = this._config.registeredModelDatabaseName;
+		let tableName = this._config.registeredModelTableName;
+		let schemaName = this._config.registeredModelTableSchemaName;
 
 		return `
-		USE [${escapedDbName}]
+		IF NOT EXISTS (
```

Comment:
So we have queries to 
1.create model table
2. predict query

Anything else?
Can we have these are reusable strings at one place rather than inline

Reply:
ideally these should move to SQLtoolsservice not here. 
I added here to make the end to end working but planning to mode them to sqltoolservice

---
In reply to: [389192425](https://github.com/microsoft/azuredatastudio/pull/9450#discussion_r389192425) [](ancestors = 389192425)

Reply:
Ah, what would be the call from ADS <--> SqlToolsService in this case? 
Also that seems like significant change.. you sure you want to refactor that in later? I don't have a preference - just asking :)

Reply:
I have to think about that but in general to avoid sql injection issues, using C# to create sql parameters works better. it's possible that this replaced with mlflow client also. in general to me this code will change alot in future

---
In reply to: [389908720](https://github.com/microsoft/azuredatastudio/pull/9450#discussion_r389908720) [](ancestors = 389908720)